boards/cypress/cy8ckit_062_ble/cy8ckit_062_ble_cy8c6347_m0_defconfig

```diff
@@ -11,3 +11,6 @@ CONFIG_BUILD_OUTPUT_HEX=y
 
 # UART driver
 CONFIG_SERIAL=y
+
+# Pinctrl driver
+CONFIG_PINCTRL=y
```

Comment:
please do not enable pinctrl at board level, make drivers select the option

Reply:
updated